src/testing/jest/jest-preprocessor.ts

```diff
@@ -136,6 +145,15 @@ export const jestPreprocessor = {
       transformOptions = jestConfigStr.config;
     }
 
+    /**
+     * At this point, `transformOptions` should be a truthy value:
+     * - if we're running Jest 27, it should have been assigned in the previous conditional
+     * - if we're running Jest 26 and under, it should have been provided to us
+     */
+    if (!transformOptions) {
+      throw 'Unable to find Jest transformation options.';
```

Comment:
This is _veryyyyy_ similar to https://github.com/ionic-team/stencil/pull/3526/files#diff-c877bce83f1eca295b644af29aac14781949edfa58775bc46b07006c8b18acabR78. I considered coalescing them into a function, but decided against it since the `transformOptions` reassignment conditional above each of them is ever so subtly different